crates/vertigo/src/dev/browser_command.rs

```diff
@@ -0,0 +1,21 @@
+use vertigo_macro::AutoJsJson;
+
+use crate::{JsJson, JsJsonContext, JsJsonDeserialize};
+
+pub fn decode_json<T: JsJsonDeserialize>(json: JsJson) -> T {
+    T::from_json(JsJsonContext::new(""), json).unwrap()
+}
```

Comment:
You can use `from_json` from `crates/vertigo/src/driver_module/js_value/serialize.rs` instead of defining a new one.